src/main/java/interfaces/Content.java

```diff
@@ -0,0 +1,13 @@
+package interfaces;
+
+import java.util.List;
+import java.util.Map;
+
+public interface Content {
+
+    Map<Integer, String>getQuestions();
+    Map<Integer, List<String>>getAnswers();
+     Map<Integer, String> getDefeatMessages();
+    public String getQuestion(Integer numberQuestion);
```

Comment:
Why not to use as argument a primitive?

Reply:
- [x] Integer type arguments  changed to int type